src/main/java/Jeremy.java

```diff
@@ -0,0 +1,48 @@
+import java.util.Scanner;
+
+public class Jeremy {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        TaskList taskList = new TaskList(100);
+        PrintUtils.greeting();
+        PrintUtils.logo();
+
+        String userInput = scanner.nextLine();
+        while (!userInput.equals("bye")) {
+            String[] parts = userInput.split(" ", 2);
+            String commandStr = parts[0];
+            String argument = parts.length > 1 ? parts[1] : "";
+            Command command = Command.fromString(commandStr);
+
+            switch (command) {
+            case LIST:
+                taskList.printList();
+                break;
+            case MARK:
+                taskList.markTaskAsDone(Integer.parseInt(argument));
+                break;
+            case UNMARK:
+                taskList.markTaskAsNotDone(Integer.parseInt(argument));
```

Comment:
Should this be blank line will be removed?

Reply:
There is no blank line in this section of code, I think again this is due to the text display on GitHub being a little weird at this part